source/core/NumericString.cpp

```diff
@@ -2596,12 +2649,25 @@ void ScanSpreadsheet(StringRef inputString, StringRef formatString, StringRef de
 VIREO_FUNCTION_SIGNATURE4(SpreadsheetStringtoArray, StringRef, StringRef, StringRef, TypedArrayCoreRef)
 {
     TypedArrayCoreRef outputArray = _Param(3);
-    if (_Param(2)->Length() ==0) {
+    if (_Param(2)->Length() == 0) {
+        STACK_VAR(String, delimiter);
+        delimiter.Value->AppendCStr("\t");
+        ScanSpreadsheet(_Param(0), _Param(1), delimiter.Value, outputArray, false);
+    } else {
+        ScanSpreadsheet(_Param(0), _Param(1), _Param(2), outputArray, false);
+    }
+    return _NextInstruction();
+}
+//-------------------------------------------------------------------
```

Comment:
Now that you're touching this function, please add a comment for the parameters

Reply:
Done.